todo_app/app.py

```diff
@@ -1,14 +1,60 @@
-from flask import Flask
+from flask import (
+    Flask,
+    render_template, 
+    request, 
+    redirect,
+    url_for,
+)
 
 from todo_app.flask_config import Config
+from todo_app.data.session_items import (
+    get_items, 
+    add_item,
+    get_item,
+    save_item,
+    delete_item,
+)
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
+# TODO: Add 'In Progress' status
+# TODO: Add date
+# TODO: Add error handling
+# TODO: Add unittests
+# TODO: Improve styling of app further
+
```

Comment:
In a real project you wouldn't keep things like this in the codebase.

TODOs in the codebase would only be if something is surprisingly lacking and ideally reference a ticket written up somewhere and maybe a reason why it's not done yet.